test/Dialect/Tpp/tpp-bufferization.mlir

```diff
@@ -590,3 +590,35 @@ func.func @fused_brgemm_op(%arg0: tensor<1x3x3xf32>, %arg1: tensor<1x3x3xf32>,
      %arg2: tensor<3x3xf32>, %arg3: tensor<3x3xf32>) -> tensor<3x3xf32>
   return %0: tensor<3x3xf32>
 } 
+
+// -----
+
+func.func @conflict_in_place(%arg0: tensor<3x3xf32>) -> tensor<3x3xf32> {
+  %0 = tpp.add(%arg0: tensor<3x3xf32>, %arg0: tensor<3x3xf32>) -> tensor<3x3xf32>
+  %1 = tpp.add(%0: tensor<3x3xf32>, %arg0: tensor<3x3xf32>) -> tensor<3x3xf32>
+  return %1 : tensor<3x3xf32>
+}
+
+// CHECK-LABEL: conflict_in_place
+// CHECK-SAME: %[[ARG:.+]]: memref<3x3xf32>
+// CHECK: %[[ALLOC:.+]] = memref.alloc() {{.+}} : memref<3x3xf32>
+// CHECK-NEXT: memref.copy %[[ARG0]], %[[ALLOC]] : memref<3x3xf32> to memref<3x3xf32>
```

Comment:
Careful, this test defines `ARG` not `ARG0`, but you check for `ARG0` below, which was defined in the previous tests.

Reply:
fixed.